doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -102,91 +319,92 @@ Submit pull request
     1000-line PR is acceptable, while a set of new tests introduced as a
     1000-line PR is not.
 
-.. _Test changes:
-
-Test changes
-~~~~~~~~~~~~
+.. _Find a reviewer:
 
-It may be helpful to read the documentation on Chapel's `Test System`_ before
-writing or modifying tests.
+Find a reviewer
++++++++++++++++
 
-* Testing your patch:
+* When you believe your pull request is ready for review, send it to a member of
+  the core Chapel team (or to the chapel-developers_ list if there isn't an
+  obvious person to review it). To make it easy to spot, please include `[PR]`
+  prior to the subject of your email.  Even the developers that have write
+  access to the Chapel repository need to have all non-trivial changes
+  reviewed. Developers who have been given write access can merge trivial
+  changes (e.g. small bug fixes, documentation changes) without review.
```

Comment:
Using the @ mention syntax in a comment on the PR is a best practice to request review from a particular person. People without commit access can't use the "Request reviewer" pane on the right. I think @ mentioning is better than emailing separately if you know who the reviewer is.

I'd like this section to be shorter. E.g.

"Once your PR is ready, you'll need to request a review. If you know who you'd like to review it, @ mention them in a comment on the PR and ask them to have a look. If you don't know who should review it, send an email to the chapel-developers_ list requesting a review and linking to the PR. Such an email should have a subject line starting with `[PR]`."

The bit starting with "Even the developers..." belongs in a "Who can commit" section in Policy Details. I don't think it's necessary here at all. The Policy Details section can say that people who have commit access can commit trivial changes without review & describe what counts as a trivial change.